package.json

```diff
@@ -51,7 +51,10 @@
     "parserOptions": {
       "ecmaVersion": 2020
     },
-    "rules": {}
+    "rules": {
```

Comment:
wat is dis

Reply:
Eslint klagde over at det ble brukt console og alert i applikasjonen, så lot den få lov.

Reply:
Fett, alltid vært irritert over det.